lib/Migration/Version10000Date20201015134000.php

```diff
@@ -220,7 +222,22 @@ public function postSchemaChange(IOutput $output, Closure $schemaClosure, array
 				->setParameter('last_mention_message', (int) $row['last_mention_message'], IQueryBuilder::PARAM_INT)
 				;
 
-			$insert->execute();
+			try {
+				$insert->execute();
+			} catch (\Exception $e) {
+				if (get_class($e) === UniqueConstraintViolationException::class) {
```

Comment:
strange approach.
no multiple catch blocks ?

Reply:
I'm not sure you can catch classes that don't exist, just wanted to be on the save side because I know `::class` works without the class existing